web/src/views/DandisetLandingView/DandisetPublish.vue

```diff
@@ -370,6 +422,15 @@ import { draftVersion, VALIDATION_ICONS } from '@/utils/constants';
 import { RawLocation } from 'vue-router';
 import { open as openMeditor } from '@/components/Meditor/state';
 
+const PUBLISH_CHECKLIST = [
+  'a descriptive title (e.g., Data related to foraging behavior in bees vs Smith et al 2022)',
+  'a description that clearly describes the dataset for another neuroscientist',
+  'a reference to a protocol in protocols.io detailing how you collected the data? this can also be another publication added as a resource.',
+  'a reference to an ethics protocol number if you have an institutional review board review the study.',
+  'funding information: dandi treats funding agencies as contributors. you can add multiple contributing institutions. unless you are the funder, you can add a new contributor of type organization, uncheck include contributor in citation, include dcite:Funder role and add the relevant award info.',
+  'reference pointers to code in github, related publications, etc.,.',
```

Comment:
Some wholesale copy editing:

```suggestion
  'A descriptive title (e.g., Data related to foraging behavior in bees vs Smith et al 2022)',
  'A clear, informative description that is helpful for other neuroscientists',
  'A reference to a protocol in protocols.io detailing how you collected the data (this can also be another publication added as a resource)',
  'A reference to an ethics protocol number if you are working with an Institutional Review Board on this study',
  'Funding information (DANDI treats funding agencies as contributors, so you can add multiple contributing institutions as needed. If you are the funder, you can add a new contributor of type "organization", uncheck "include contributor in citation", set the role as "dcite:Funder", and include the relevant award information)',
  'References to code in GitHub, related publications, etc.',
```

Reply:
i think this one conflicts with the previous commit (the first bullet). otherwise the changes LGTM.